src/oop/special_class_method/hard/q13/QuestionText.md

```diff
@@ -0,0 +1,37 @@
+# oop-special_class_method-hard-q13
+
+## Question Text
+
+Suppose you were hired by a departmental store to digitize their catalogue system, and one of the classes you are making
+for objects is a **Cup** class. This class has three attributes called **liquid**, **color** , and **volume** which
+cannot be modified, but can be accessed
```

Comment:
This says the three attributes have getters but not setters, but everywhere else has a liquid setter and a volume getter. I think the one described here makes more sense because it gives a use for every variable

Reply:
Its a parsons so I think for them to make 3 same type methods would be redundant and making one of each test their abilities better.